afs-core/src/main/java/com/powsybl/afs/AppFileSystemTool.java

```diff
@@ -99,6 +106,11 @@ public Options getOptions() {
                         .desc("archive dependencies")
                         .hasArg(false)
                         .build());
+                topLevelOptions.addOption(Option.builder("keepResults")
+                        .longOpt(RESULT)
+                        .desc("keep results of MS and metrix")
```

Comment:
MS and metrix objetcs don't exist here. 
Also this should not be a top level option.
The option should be to clear the result and not keep them, in order to have the same behavior as before.